jcommune-view/jcommune-web-view/src/main/resources/org/jtalks/jcommune/web/view/messages_en.properties

```diff
@@ -493,3 +493,7 @@ label.registration.success.2=s
 
 label.search.user=Search user
 label.search.user.empty=No users found
+label.user.groups=Groups
+label.user.group.add.error=Error on add group
+label.user.group.delete.error=Error on delete group
+label.user.search.connection.lost=Connection to the server was lost
```

Comment:
such key already exists:
label.connection.lost=Connection to the server was lost, please save your text locally.
maybe label.connection.lost.error ?